sdk/messaging/azservicebus/sender.go

```diff
@@ -53,18 +45,18 @@ func (s *Sender) NewMessageBatch(ctx context.Context, options *MessageBatchOptio
 		return nil, err
 	}
 
-	maxBytes := int(sender.MaxMessageSize())
+	maxBytes := int32(sender.MaxMessageSize())
```

Comment:
Let me check what the actual native type is. It might be that I need to go the other way, even if, in reality, the domain doesn't support a crazy large max message size.

Reply:
So this is an AMQP field (straight from the attach frame) and it's typed to be a ulong, not a long.

* http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-transport-v1.0-os.html#:~:text=See%20subsection%202.6.7.-,max-message-size,-the%20maximum%20message
* http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-types-v1.0-os.html#type-ulong

So we should probably follow suit there, but int32 is definitely wrong.